src/FWlib/peripheral_adc.h

```diff
@@ -165,6 +165,14 @@ typedef struct
  */
 void ADC_ClkCfg(SADC_adcClk clk);
 
+/**
+ * @brief Select DMA request channel of transmitting ADC data
+ *
+ * @param[in] sel             DMA request channel(0-7)
+ * @return                    null
+ */
+void ADC_SetDmaSel(uint8_t sel);
+
```

Comment:
Delete this API. Instead,

1. `SYSCTRL_SelectUsedDmaItems`;
2. `SYSCTRL_GetDmaId`